doc/HOWTO_RELEASE.rst.txt

```diff
@@ -40,16 +40,17 @@ Release Scripts
 
 Supported platforms and versions
 ================================
-`NEP 29`_ outlines which Python versions are supported; For the first half of
-2020, this will be Python >= 3.6. We test NumPy against all these versions
-every time we merge code to master.  Binary installers may be available for a
-subset of these versions (see below).
+:doc:`NEP 29<neps:nep-0029-deprecation_policy>` outlines which Python versions
```

Comment:
```suggestion
:ref:`nep0029` outlines which Python versions
```
The links/abbreviations can be made shorter with intersphinx I think. This will replace the `:ref:` with the full title of the document.